docs/docs/glossary/npm.md

```diff
@@ -0,0 +1,59 @@
+---
+title: npm or Node package manager
+disableTableOfContents: true
+---
+
+Learn what _npm_ is, how to use it, and how it fits in to the Gatsby ecosystem.
+
+## What is npm?
+
+<abbr>
+  npm
+</abbr>, or Node package manager, is the default package manager for the [Node.js](/docs/glossary/node) JavaScript runtime. It lets you install and update libraries and frameworks (dependencies) for Node-based projects, and interact with the npm Registry. You'll use npm to install and upgrade Gatsby and its plugins.
+
+npm is a [command line](/docs/glossary#command-line) tool. You'll need Terminal (Mac, Linux) or Command Prompt (Windows) in order to run its commands. To use one of npm's features, type `npm <command>` . For example, `npm help` displays a list of available features, including `install`, `uninstall`, `update`, and `search`.
+
+npm is installed alongside Node during the default [installation process](/tutorial/part-zero/#install-nodejs-for-your-appropriate-operating-system). You don't need to take any additional steps to add it to your environment.
+
+### Using npm to install Gatsby
+
+It's recommended to install Gatsby globally, so that you can use Gatsby as a CLI application. To do so, use `npm install` with the `--global` or `-g` flags.
+
+```shell
+npm install -g gatsby-cli
+```
+
+Once the installation completes, you can run `gatsby new my-project` to create a new Gatsby project.
+
+### Using npx to install Gatsby
+
+> **Note:** `npx` requires npm version 5.2 or later. If you've installed the latest versions of Node and NPM, you should also have npx. Otherwise, you should upgrade Node and/or npm.
```

Comment:
```suggestion
> **Note:** `npx` requires npm version 5.2 or later. If you've installed the latest versions of Node and npm, you should also have npx. Otherwise, you should upgrade Node and/or npm.
```